frontend/src/components/interviewCard.js

```diff
@@ -27,20 +27,30 @@ class InterviewCard extends Component {
   }
 
   render() {
+    const closeBtn = (
+      <button className="close" onClick={() => this.toggle()}>
+        &times;
+      </button>
+    )
+
     return (
       <>
         <Modal isOpen={this.state.modal} toggle={() => this.toggle()}>
-          <ModalHeader>
+          <ModalHeader close={closeBtn}>
             Are you sure you want to delete this interview? You won't be able to get it back.
           </ModalHeader>
           <ModalFooter>
             <Button onClick={() => this.toggle()}>Return</Button>
-            <Button onClick={this.handleDeleteClick}>Delete</Button>
+            <Button color="danger" onClick={this.handleDeleteClick}>
+              Delete
+            </Button>
           </ModalFooter>
         </Modal>
         <Card className="candidate-card">
           <CardBody>
-            <CardTitle>Overall Interview Score: {this.props.overallScore}</CardTitle>
+            <CardTitle onClick={() => this.handleViewDetails()}>
```

Comment:
why anonymous?

Reply:
Yeah, can be simplified to `onClick={this.handleViewDetails}`

Reply:
yep, just realized a couple of these functions were not binded to the class, which was why the anonymous way worked